apps/teamsjs-cc-app/tabs/src/helpers/index.ts

```diff
@@ -0,0 +1,3 @@
+export const booleanToString = (value: boolean) => {
+    return value ? 'Yes' : 'No';
```

Comment:
nitpick: not sure if it's important that this is 'Yes' and 'No' instead of 'true' and 'false', but booleans should turn into strings without needing to do anything fancy.

Reply:
Honestly, if we removed this, we would need to write a similar logic for everything in the matrix table. This simplifies the code a decent amount unless you know a way we can do this easier.